teacher.rb

```diff
@@ -1,8 +1,8 @@
-require_relative('./person')
+require_relative 'person'
 
 class Teacher < Person
-  def initialize(age, specialization, name = 'Unknown', parent_permission: true)
-    super(age, name, parent_permission)
+  def initialize(age, specialization, parent_permission: true, name: 'Unknown')
+    super(age, parent_permission: parent_permission, name: name)
     @specialization = specialization
```

Comment:
- [ ] Well done for fixing the bug 🐛 with creating a `Student` or a `Teacher` 👏
Having said that, we have to note that `parent_permission` is not required to initialize a new `Teacher`. As you can see in the [video](https://www.youtube.com/watch?v=vkkgrhD6aXQ) screenshot below, all we need are: `age`, `name`, and `specialization`. Kindly make this change 🙏 
## Video Screenshot
![image](https://github.com/danielmatama/OOP-library/assets/46839548/2f3d559e-86a5-490f-8387-5ab51ff3a41f)
## Your Implementation
![image](https://github.com/danielmatama/OOP-library/assets/46839548/07d21ac1-3da1-4479-9968-b00d9502bb40)